index.html

```diff
@@ -10,7 +10,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0, minimum-scale=1.0"
     />
-    <link rel="stylesheet" href="./styles/custom.css" />
+    <!-- <link rel="stylesheet" href="./styles/custom.css" /> -->
```

Comment:
@geoffreylgv why did you comment this line?

Reply:
We found that the drop-down appears when it’s commented @antonio-pedro99